### ISSUE TITLE
Aliens are instantly husked upon death from burn damage

### ISSUE THREAD

**Issue by carlarctg** [tgstation/tgstation#87914]
### Client Version:

n/a

### Issue Summary:

Xenomorphs behave rather uniquely compared to humans, especially in their health department. Various stupid oddities in it result in the fact that xenomorphs almost always husk nigh-instantly after death - they have 300+ burn or whatnot, when a normal human has 200 burn at the point of death. This is annoying for some niche interactions, such as reviver implant, which works on xenos, never being able to do its job because it can't revive husks.

That's the walance bit. The other bit is that, uh, how do you husk an exoskeleton. I dunno seems odd.

TL;DR: It arises from buggy behavior, breaks niche interactions, and makes no 'realistic' sense anyways.

i tried fixing in #87771 but the husk code is shit, the alien code is shit, my brain is shit, we are all shit on this blessed daty

### Round ID:

n/a

### Test Merge Information:

n/a

### Reproduction Steps:

1. shoot alien to death
2. crispy
